Update Get-DatabricksDBFSFile.ps1

bilalAchahbar opened the pull request:
I can't see exactly why you added the PS version required to 6 or higher. But this provided with some errors inside PS version 5.1. 
reference:  https://github.com/PowerShell/PowerShell/issues/3131
Because I run this script inside Azure DevOps pipelines (which has no possibility to use PS 6 or higher).
I changed the script to handle the commands and rest call to capture the file even with PS 5.1.
If you find another way around this no problem, just pointing out the problem and where the workaround should be handled.
*Also remember your scripts are being used by people in automated flaws, changing them will have some impact.

simondmorias commented:
Thanks for the PR. I think the issue with pre v6 was the line:
```
Set-Content -Path $TargetFile -Value $finalFile -AsByteStream
```

I didn't know about that option you have used.

Is there any reason you have changed the function name (added the s to the end)? That's breaking the tests.

If you can put the function name back I can test again and push through.

Thanks for your contribution.

bilalAchahbar commented:
I changed the function and used it inside my own script.
But because I already use other functions of your PS module I've added the "s" to avoid conflict with your "Get-DatabricksDBFSFile" function.
It was a copy paste mistake , my bad.

Also I've changed the Set-Content, but also fixed the body, I saw you've set the body to a json type. In PS 5 this will eventually mean that the Content type will be overrules by the type of the body, and the script will fail because the content type "application/json" is not accepted with the databricks api call. So I changed the type of the body to fix that issue.
